odoo/addons/test_orm/tests/test_new_fields.py

```diff
@@ -30,18 +29,18 @@ def setUp(self):
         # for tests methods that create custom models/fields
         self.addCleanup(self.registry.reset_changes)
         self.addCleanup(self.registry.clear_all_caches)
-        super(TestFields, self).setUp()
-        self.env.ref('test_new_api.discussion_0').write({'participants': [Command.link(self.user_demo.id)]})
+        super(__class__, self).setUp()
```

Comment:
shouldn't we use
```suggestion
        super().setUp()
```
?